sw/host/opentitanlib/src/app/config/structs.rs

```diff
@@ -135,6 +136,12 @@ pub struct ConfigurationFile {
     /// List of names of other configuration files to include recursively.
     #[serde(default)]
     pub includes: Vec<String>,
+    ///
```

Comment:
Do you have a sample config file you could include in this PR so I can see a proposed working example?

Reply:
I have added three built-in configuration files to this CL, demonstrating how to declare which kind of chip under test is accessible in each case.

The GSC team would then want to add a file on top, to declare the ChromeOS signal names, such as:
```
{
  "pins": [
    {
      "name": "AC_PRESENT",
      "mode": "OpenDrain",
      "level": true,
      "pull_mode": "PullUp",
      "alias_of": "IOR2"
    },
    /* Many more pins, with default levels. */
  ],
  "requires": {
    "chip": "opentitan"
  }
}
```

This states that when OpenTitan is used in a Chromebook, the pin IOR2 will be used for signalling whether a charger i plugged in.  Using this file only makes sense when the testbed (`opentitansession`) has OpenTitan (and hence already uses `hyperdebug_cw310.json`, which includes `opentitan.json`).  We have other testbeds, with other GSC chips that have other signal names and mappings.  If the above file was accidentally used with one of those testbeds, prior to this change, the best case scenario was to get a "Unknown pin IOR2", at the first time a test case tried to simulate plugging or unplugging a charger, and the worst case is that the other chip happens to have a pin called "IOR2", but it is used for something other than "AC_PRESENT", in which case the test script would only fail later.

With this change, the mistake of running a OpenTitan test script against a non-OpenTitan testbed would be discovered immediately upon `opentitantool` connecting to the running `opentitansession`.